src/canvas-view/canvas/node-toolbar.component.js

```diff
@@ -0,0 +1,34 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .component('nodeToolbar', {
+      templateUrl: 'canvas-view/canvas/node-toolbar.html',
+      bindings: {
+        node: '=',
+        nodeActions: '=',
+        nodeClickHandler: '<', // @callback args - (action {string}, node {Object})
```

Comment:
We sure about calling them arguments... what about parameters? 

Also it may be a good idea to be consistent and apply similar comments to other callbacks that pass parameters back.

Reply:
I thought args/params were pretty much the same thing? I actually documented the nodeClickHandler in ng-docs so might not need this comment here.

Reply:
reworded the ng-docs to encompass the concept of both arguments and parameters for the nodeClickHandler in  the ```canvas.js``` example. Do we still need the comment?